src/main/java/com/gd/clinic/entity/Prescription.java

```diff
@@ -0,0 +1,56 @@
+package com.gd.clinic.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.gd.clinic.model.NewPrescriptionDto;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.*;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "prescriptions")
+public class Prescription {
```

Comment:
also you need to add `equals` and `hashCode` methods
usually they are generated by `@Data` lombok annotation, but it's not recommended to use with Hibernate entities

Reply:
Fixed in previous comment, those methods are now overriden by all entities.